### DISCUSSION TITLE
React voice to Text component

### DISCUSSION THREAD

**Discussion by Thichman** [vercel/community#2668] [Help]
### Summary

I am trying to create a voice to text implementation into my app. But all of the libraries I am finding have dependencies on react 16 while my project is version 18.2. Does anyone know a library I can use in order to have this work in the new version of react. 

### Steps to Reproduce

Try and download any of the voice to text libraries like https://www.npmjs.com/package/react-speech-recognition?activeTab=readme into a react 18 project.

**Comment by z4nr34l** [comment]
Based on my current projects, such libraries - despite throwing warnings to the console - just work.

If you have a problem, please share more details so I can understand it better and try to help.

**Reply by Thichman** [reply]
This is the main error I get when actually trying to install the package. 
<img width="911" alt="image" src="https://github.com/vercel/community/assets/55109754/2dfa686e-2558-457f-aa45-0e2fd991847d">
It is always the same dependency error. It cannot resolve the dependency tree which I am not 100% sure what that means. This is the full error: 
<img width="825" alt="image" src="https://github.com/vercel/community/assets/55109754/bc66de5b-9e2e-430d-9c5d-985647a0c995">

This is in the new react 18.2 projects that I am trying to build. Thank you for the help!
